graphene_linked_events/schema.py

```diff
@@ -715,15 +719,23 @@ class ImageMutationResponse(ObjectType):
     result_text = String()
 
 
+def _validate_image_upload(image):
+    if image.size > settings.MAX_UPLOAD_SIZE:
+        raise UploadImageSizeExceededError(
+            f"Upload file size cannot greater than {settings.MAX_UPLOAD_SIZE} bytes"
+        )
+
+
 class UploadImageMutation(Mutation):
     class Arguments:
         image = UploadImageMutationInput()
 
     response = Field(ImageMutationResponse)
 
-    @staff_member_required
+    # @staff_member_required
```

Comment:
Hmm, intentional?

Reply:
ugh.. forgot to ctrl + Z it after testing 😅

Reply:
That would have been my first guess 🙂